products/BellHybrid/alarms/src/actions/FrontlightAction.cpp

```diff
@@ -3,30 +3,33 @@
 
 #include "FrontlightAction.hpp"
 #include "BellAlarmConstants.hpp"
-
 #include <service-evtmgr/ScreenLightControlMessage.hpp>
 #include <service-evtmgr/ServiceEventManagerName.hpp>
-
 #include <service-db/Settings.hpp>
 #include <db/SystemSettings.hpp>
+#include <module-utils/utility/FrontlightUtils.hpp>
 #include <module-utils/utility/Utils.hpp>
 
 namespace alarms
 {
     namespace
     {
-        constexpr std::string_view alarmFrontlightOff     = "0";
-        constexpr std::string_view prewakeupFrontlightOff = "0";
+        constexpr auto alarmFrontlightOff     = std::string_view{"0"};
+        constexpr auto prewakeupFrontlightOff = std::string_view{"0"};
 
         /* Reach and maintain target brightness 1 minute faster than
          * the duration of the action. */
         constexpr auto frontlightRampDurationOffset = std::chrono::minutes{1};
         constexpr auto initialTargetDuration        = std::chrono::seconds{5};
-        constexpr auto initialZeroBrightnessValue   = 10;
+#if defined(CONFIG_VERSION_PRO) && (CONFIG_VERSION_PRO == 1)
+        constexpr auto initialZeroBrightnessValue = 5.0f;
+#else
+        constexpr auto initialZeroBrightnessValue = 10.0f;
+#endif
 
-        void validateBrightness(std::string &brightness)
+        inline void validateBrightness(std::string &brightness)
         {
-            constexpr auto defaultBrightness = std::string_view{"50.0"};
+            constexpr auto defaultBrightness = std::string_view{"5"};
```

Comment:
why are we changing the default value?

Reply:
I changed the way brightness setting is stored in the database, previously PWM percentage was stored, now it's 1-10 scale used by the OS. 50% PWM is the same as 5 in this scale